docs/user/openstack/install_upi.md

```diff
@@ -156,25 +148,42 @@ python3 -m pip install --upgrade pip
 python3 -m pip install yq openstackclient openstacksdk netaddr
 ```
 
+Make sure, that none of copied playbooks contains an alias for ansible
+utility filters:
+
+```sh
+for playbook in *.yaml; do
+    sed -i -e 's/ansible.utils.//g' $playbook
+done
+```
+
 ### Fedora
 
 This command installs all required dependencies on Fedora:
 
 ```sh
-sudo dnf install python3-openstackclient ansible-core python3-openstacksdk python3-netaddr
+sudo dnf install python3-openstackclient python3-netaddr ansible-collections-openstack ansible-collection-ansible-utils
 ```
 
 [ansible-upi]: ../../../upi/openstack "Ansible Playbooks for Openstack UPI"
 
 ## Ansible Collections
 
-The Ansible Collections are not packaged (yet) on recent versions of OSP and RHEL when `ansible-core` is
-installed instead of Ansible 2.9. So the collections need to be installed from `ansible-galaxy`.
+Alternatively you could use ansible-galaxy to install required collections,
+although note, there might be incompatibilities as for example starting from
+`openstack.cloud-2.x.x` release will require `openstacksdk` with version at
+least 1.0.0. Ignoring that will eventually make some playbooks will fail later
+on.
 
 ```sh
-ansible-galaxy collection install "openstack.cloud:<2.0.0" ansible.utils community.general
+ansible-galaxy collection install openstack.cloud ansible.utils community.general
 ```
 
+You might want to pin version of `opestack.cloud` prior to version 2.x.x with
+`openstack.cloud:<2.0.0`, although this path is untested. Installing collections
+using ansible-galaxy is meant for controlled environments like Python
+virtualenv.
```

Comment:
I am a bit confused by this part. Can you please make it clear what is required, and what is supported?